avm/res/api-management/service/identity-provider/main.bicep

```diff
@@ -14,6 +14,14 @@ param authority string = ''
 @description('Conditional. Client ID of the Application in the external Identity Provider. Required if identity provider is used.')
 param clientId string = ''
 
+@description('Optional. The client library to be used in the developer portal. Only applies to AAD and AAD B2C Identity Provider.')
```

Comment:
@anotherRedbeard for consistency, let's not allow an empty string as a default value. Instead, let's `null` this if it is not set.

Reply:
@tony-box, makes sense. These changes have been applied.